### DISCUSSION TITLE
Scanned pdf

### DISCUSSION THREAD

**Discussion by erkin98** [jsvine/pdfplumber#380] [Q&A] [answered]
is there any options for read and parse scanned pdfs?i tried one of them and it returns none.i think this library doesnt support for scaaned pdfs

**Comment by samkit-jain** [comment, marked as answer]
Hi @erkin98 Yes, you are correct. Pdfplumber does not work on scanned PDFs. You may try using tools like [Tesseract](https://github.com/tesseract-ocr/tesseract) to convert a scanned PDF into a text-based PDF and then use pdfplumber on it.
